packages/platforms/react-native/ios/BugsnagReactNativePerformance.mm

```diff
@@ -82,12 +82,118 @@ @implementation BugsnagReactNativePerformance
     return hexStr;
 }
 
- RCT_EXPORT_METHOD(requestEntropyAsync:(RCTPromiseResolveBlock)resolve
-                   rejecter:(RCTPromiseRejectBlock)reject)
- {
-     NSString *hexStr = getRandomBytes();
-     resolve(hexStr);
- }
+RCT_EXPORT_METHOD(requestEntropyAsync:(RCTPromiseResolveBlock)resolve
+                   rejecter:(RCTPromiseRejectBlock)reject) {
+    NSString *hexStr = getRandomBytes();
+    resolve(hexStr);
+}
+
+RCT_EXPORT_BLOCKING_SYNCHRONOUS_METHOD(getNativeConstants) {
+    NSMutableDictionary *nativeDirs = [NSMutableDictionary new];
+    nativeDirs[@"CacheDir"] = NSSearchPathForDirectoriesInDomains(.cachesDirectory, .userDomainMask, true).first!;
```

Comment:
Even though we don't expect the result to be empty, I think that adding a null check instead of a force unwrap would be a better option here

Reply:
changed